extensions/keyboard-maestro/package.json

```diff
@@ -6,12 +6,13 @@
   "icon": "icon.png",
   "author": "eluce2",
   "contributors": [
-    "suaw3k"
+    "suaw3k",
+    "fakerr"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "list",
+      "name": "list-macros",
```

Comment:
Can you use the same name, this is the unique key 🙂

Reply:
No worries, I just swapped it back.